lego/apps/events/tasks.py

```diff
@@ -210,11 +325,19 @@ def check_for_bump_on_pool_creation_or_expansion(self, event_id, logger_context=
 
 @celery_app.task(serializer="json", bind=True, base=AbakusTask)
 def stripe_webhook_event(self, event_id, event_type, logger_context=None):
+    """
+    Task that handles webhook events from Stripe, and updates the users registration in accordance
+    with the payment status.
+    """
     self.setup_logger(logger_context)
 
-    if event_type in ["charge.failed", "charge.refunded", "charge.succeeded"]:
+    if event_type in ["payment_intent.payment_failed", "payment_intent.succeeded"]:
```

Comment:
I would prefer if these were constants and not hardcoded here.

Reply:
:ok: